{"error":{"code":"1","message":"Acesso não autorizado","exception":{"ClassName":"Vtex.Practices.ServiceModel.Client.Exceptions.UnauthorizedException","Message":"Acesso não autorizado","Data":null,"InnerException":null,"HelpURL":null,"StackTraceString":"   at Vtex.Commerce.Oms.WebApi.Filter.ServiceAuthorizationFilterAttribute.<OnAuthorizationAsync>d__0.MoveNext() in F:\\BuildAgent\\work\\96a083434c98702d\\src\\services\\Private Assemblies\\Vtex.Commerce.Oms.WebApi\\Filter\\ServiceAuthorizationFilterAttribute.cs:line 74\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Filters.AuthorizationFilterAttribute.<ExecuteAuthorizationFilterAsyncCore>d__2.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()","RemoteStackTraceString":null,"RemoteStackIndex":0,"ExceptionMethod":"8\nMoveNext\nVtex.Commerce.Oms.WebApi, Version=1.5.170.996, Culture=neutral, PublicKeyToken=null\nVtex.Commerce.Oms.WebApi.Filter.ServiceAuthorizationFilterAttribute+<OnAuthorizationAsync>d__0\nVoid MoveNext()","HResult":-2146233088,"Source":"Vtex.Commerce.Oms.WebApi","WatsonBuckets":null}}}

